### PR TITLE
Add method for debugging + minimal refactoring

### PR DESCRIPTION
**Pull request by rah-emil**
Please, merge this editions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +69,15 @@ module.exports = {
 };
 ```
 
+#### Global store (`~/store/index.js`)
+```javascript
+import createCache from '../plugins/@rah-emil-vuex-cache';
```
Comment (VitorLuizC) [inline README.md:74]:
```suggestion
import createCache from 'vuex-cache';
```

**Review by VitorLuizC — CHANGES_REQUESTED** [review]
I'll be glad to merge your PR if you undo formatting changes and the replacement of `.toBe(false)` and `.toBe(true)` assertions. We choose `.toBe(false)` assertion because it doesn't considers `0`, `""`, `NaN`, `null` and `undefined` values as valid, and the same applies to `.toBe(true)`.

File: README.md (hunk 1 above)
Comment (rah-emil) [reply README.md:74]: Came here by accident. Thanks!)

**Comment by rah-emil** [comment]
I hope everything is fine😊

**Review by VitorLuizC — APPROVED** [review]

**Comment by VitorLuizC** [comment]
Released under [`v3.5.0`](https://www.npmjs.com/package/vuex-cache/v/3.5.0) version.

**Pull request merged**